test/about/historyTest.js

```diff
@@ -81,10 +81,11 @@ describe('about:history', function () {
 
     it('opens a new tab with the location of the entry when double clicked', function * () {
       const site = Brave.server.url(browseableSiteUrl)
+      const target = 'table.sortableTable td.title[data-sort="' + browseableSiteTitle + '"]'
```

Comment:
Do we need to change this? Because we try to use data attributes, so that we are not binned to the html elements